requirements/edx/base.txt

```diff
@@ -47,7 +47,8 @@ meliae==0.4.0
 mongoengine==0.10.0
 MySQL-python==1.2.5
 networkx==1.7
-nose==1.3.7
+git+https://github.com/edx/nose.git@99c2aff0ff51bf228bfa5482e97e612c97a23245#egg=nose==1.3.7
```

Comment:
Good point. Yes we do. I'll have to include that with any announcement.
I also forgot to point out that we are installing nose here, instead of github.txt, because we add so many plugins, some of which surely have nose as a prereq. This means by the time we get to github.txt, it will be a no-op.